### PR TITLE
Update replies.go

### PR DESCRIPTION
**Pull request by DoctorBang**
Nice gif for 1.16, as well as the download for Legacy versions of Impact.

### CODE REVIEW

File: replies.go [hunk 1]
```diff
@@ -161,10 +161,10 @@ var replies = []Reply{
 	},
 	{
 		pattern: `(1\.16.*?(update|coming|support|release|impact|version|eta|when|out|support)|(update|coming|support|release|impact|version|eta|when|out|support).*?1\.16)`,
-		message: "No ETA on 1.16 Impact release, a message will be posted in <#" + announcements + "> when development starts & nightly builds.",
+		message: "No ETA on 1.16 Impact release, a message will be posted in <#" + announcements + "> when development starts & nightly builds. And please: https://cdn.discordapp.com/attachments/617140506069303306/741921114145685544/stfu30.gif",
 	},
 	{
 		pattern: `(impact.+(1\.8|1\.7))|((1\.8|1\.7).impact)`,
-		message: "Impact for older versions is no longer availible to comply with Mojang's EULA.",
+		message: "Impact for older versions is no longer availible to comply with Mojang's EULA. You can still download them from: https://snorlax.cc/impact.html",
```
Comment (LeafHacker) [inline replies.go:168]: We cannot provide or condone illegally distributions of Minecraft/optifine/old-impact.

File: replies.go [hunk 2]
```diff
@@ -161,10 +161,10 @@ var replies = []Reply{
 	},
 	{
 		pattern: `(1\.16.*?(update|coming|support|release|impact|version|eta|when|out|support)|(update|coming|support|release|impact|version|eta|when|out|support).*?1\.16)`,
-		message: "No ETA on 1.16 Impact release, a message will be posted in <#" + announcements + "> when development starts & nightly builds.",
+		message: "No ETA on 1.16 Impact release, a message will be posted in <#" + announcements + "> when development starts & nightly builds. And please: https://cdn.discordapp.com/attachments/617140506069303306/741921114145685544/stfu30.gif",
 	},
```
Comment (LeafHacker) [inline replies.go:165]: I wonder if using `alt text` syntax is better? At the very least a descriptive link is better than just pasting a massive URL into the message.

Perhaps the better solution is to add a way for bot replies to have an image attached, this wouldn't be particularly complex.

**Review by LeafHacker — CHANGES_REQUESTED** [review]
Not sure about this

**Comment by DoctorBang** [comment]
Honestly this PR was for the meme. And if you looked at the link, you'd see that the link to Impact 1.8 was a rick-roll Leaf.

**Comment by LeafHacker** [comment]
lmao

**Pull request closed without merging**